backend/dataall/core/organizations/services/organization_service.py

```diff
@@ -305,6 +305,13 @@ def resolve_organization_by_env(uri):
         context = get_context()
         with context.db_engine.scoped_session() as session:
             env = EnvironmentRepository.get_environment_by_uri(session, uri)
+            ResourcePolicyService.check_user_resource_permission(
```

Comment:
We could just call `dataall.core.organizations.services.organization_service.OrganizationService.get_organization`